recaf-ui/src/main/resources/style/seemless-list-view.css

```diff
@@ -0,0 +1,7 @@
+.indexed-cell:even {
+    -fx-background-color: rgb(50, 50, 50);
+}
+
+.indexed-cell:odd {
+    -fx-background-color: rgb(50, 50, 50);
+}
```

Comment:
Generally unique components all of a single type get put into a single css file _(Note how the `text.css` covers a variety of cases)_.

So this would instead go into `list.css` instead of being its own file.